utils/hct/hctdb.py

```diff
@@ -1427,6 +1427,19 @@ def build_semantics(self):
             (5, "Invalid", ""),
             ])
 
+        FPDenormMode = db_dxil_enum("FPDenormMode", "Floating point behavior", [
+            (0, "Any", "Undefined behavior for denormal numbers"),
+            (1, "Preserve", "Preserve denorm input. Flush denorm output to zero"),
```

Comment:
See comment above about mismatch between flags and descriptions, obviously this is the place to fix it...